frontend/src/lib/services/Libre311/Libre311.ts

```diff
@@ -238,15 +290,6 @@ export interface Open311Service {
 	getServiceRequest(params: HasServiceRequestId): Promise<ServiceRequest>;
 }
 
-// https://wiki.open311.org/GeoReport_v2/
-export interface Open311Service {
-	getServiceList(): Promise<GetServiceListResponse>;
-	getServiceDefinition(params: HasServiceCode): Promise<ServiceDefinition>;
-	createServiceRequest(params: CreateServiceRequestParams): Promise<CreateServiceRequestResponse>;
-	getServiceRequests(params: GetServiceRequestsParams): Promise<ServiceRequestsResponse>;
-	getServiceRequest(params: HasServiceRequestId): Promise<ServiceRequest>;
-}
-
```

Comment:
was this suppose to be removed?

Reply:
I removed it since it was exactly identical to the Open311Service directly above. Figured there's no reason to have two identical items right next to each other.